Swoffm/fossilhunt
# Gs home

# Description
Users now have a home page / landing page to view new information about the website and view updates
Fixes # (issue)
## Type of change
# Testing Instructions
Make sure user's can switch between home page and click on the logo in on the nav. Log In (has no functionality) 
# Checklist:
